### PR TITLE
chore: create MySQL quickstart kit

### PR DESCRIPTION
**Pull request by aliandersonsn**
Search Nurture's recommendations for MySQL quickstart copy

## Summary

A concise description of the changes being introduced.
If this is a new Observability Pack, please provide a description of its use cases and a list of the Pack's components.

### Screenshots

Images of any visual changes, such as a dashboard.

### CODE REVIEW

File: packs/mysql/config.yml [hunk 1]
```diff
@@ -4,8 +4,31 @@ name: mysql
 
 # Description of the pack
 description: |+
-  Official New Relic pack for MySQL and MariaDB
-  Use this pack together with the New Relic MySQL On Host Integration to get insight into the performance of your MySQL and MariaDB instances.
+  # MySQL Performance Monitoring
+
+  Applications powered by relational database management systems demand the user to understand how the application uses it. Quickly identify and resolve the source server issues with MySQL performance monitoring tools.
```
Comment (jpvajda) [inline packs/mysql/config.yml:9]: @bradleycamacho tagged. I feel these are too wordy....  they read like marketing content but if this is the approach we are taking with these descriptions for SEO reasons, i'll stop commenting. 😄

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: packs/mysql/config.yml (hunk 1 above)
Comment (bradleycamacho) [reply packs/mysql/config.yml:9]: I cut down the content for clarity.

**Comment by bradleycamacho** [comment]
I've made final copy edits for brevity and to make the instant value of the quickstart more clear. Should be ready to merge.

**Review by bradleycamacho — APPROVED** [review]

**Comment by jpvajda** [comment]
fixed merge conflicts

**Review by polfliet — APPROVED** [review]
